source/features/hidden-review-comments-indicator.css

```diff
@@ -49,4 +49,4 @@
 
 .file-diff-split .rgh-comments-indicator button {
 	padding-right: 1px;
-}
+}
```

Comment:
Strangely this was not caught by style lint. Every file must end with a line break